src/lucky_migrator/migration.cr

```diff
@@ -92,7 +104,10 @@ abstract class LuckyMigrator::Migration::V1
   private def execute_in_transaction(statements : Array(String))
     DB.open(LuckyRecord::Repo.settings.url) do |db|
       db.transaction do |tx|
-        statements.each { |s| tx.connection.exec s }
+        statements.each do |s|
+          tx.connection.exec set_schema unless @@schema == "public"
```

Comment:
Is the schema set temporarily for the query and then returned to "public"? Because I feel like this line of code will change the schema but never set it back to "public".

Reply:
Oh this is a class variable, when task finishes, it will get back to public, if not, the user that changed it should bring it back to public. The set schema stament only changes the schema for the current session (connection).